DiscordChatExporter.Domain/Exporting/ExportRequest.cs

```diff
@@ -81,6 +83,27 @@ public partial class ExportRequest
             Snowflake? after = null,
             Snowflake? before = null)
         {
+
+            // Formats path
+            outputPath = Regex.Replace(outputPath, "%.", m =>
```

Comment:
```suggestion
            outputPath = Regex.Replace(outputPath, "%\w", m =>
```

A bit less aggressive pattern. Putting catch-alls (`.`) scares me for some reason. :D

Reply:
This no longer catches `%%`. We could add a special case for that, but I don't think the catch-all is a problem since most patterns are simply ignored due to the `_ => m.Value` case.

Reply:
Ah, I see. Good point.